packages/eslint-config-airbnb-base/package.json

```diff
@@ -72,21 +71,22 @@
     "babel-preset-airbnb": "^4.5.0",
     "babel-tape-runner": "^3.0.0",
     "eclint": "^2.8.1",
-    "eslint": "^7.32.0 || ^8.2.0",
-    "eslint-find-rules": "^4.1.0",
+    "eslint": "^9.8.0",
     "eslint-plugin-import": "^2.30.0",
     "in-publish": "^2.0.1",
-    "safe-publish-latest": "^2.0.0",
-    "tape": "^5.9.0"
+    "safe-publish-latest": "^2.0.0"
   },
   "peerDependencies": {
-    "eslint": "^7.32.0 || ^8.2.0",
+    "eslint": "^9.8.0",
```

Comment:
this is a breaking change. please limit yourself to things that work with eslint 7+.